src/core.js

```diff
@@ -146,7 +146,7 @@ config.modules.push( config.currentModule );
 // Initialize more QUnit.config and QUnit.urlParams
 (function() {
 	var i, current,
-		location = window.location || { search: "", protocol: "file:" },
+		location = global.location || { search: "", protocol: "file:" },
```

Comment:
We should leave this as a check on `window`, since we can assume that the `location` property only exists there, just like we do for `document`.